Trimble.Modus.Components/Controls/ListView/TextCell.xaml

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ViewCell xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:converters="clr-namespace:Trimble.Modus.Components.Converters"
+             xmlns:constant="clr-namespace:Trimble.Modus.Components.Constant"
+             xmlns:helpers="clr-namespace:Trimble.Modus.Components.Helpers"
+             x:Name="textCell"
+             x:Class="Trimble.Modus.Components.TextCell">
+  <Grid x:Name="grid" >
+    <Grid.Resources>
+      <ResourceDictionary>
+        <converters:ImageNullOrEmptyBoolConverter x:Key="ImageNullOrEmptyBoolConverter" />
+        <converters:StringNotNullOrEmptyBoolConverter x:Key="StringNotNullOrEmptyBoolConverter" />
```

Comment:
Use community toolkit's converters wherever possible
https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/converters/is-string-null-or-empty-converter

Reply:
Done